rho/facts.py

```diff
@@ -202,6 +204,8 @@ def new_fact(fact_name, description, is_default=None,
 new_fact('jboss.fuse-on-karaf.find-karaf-jar',
          'Use the find utility to locate karaf.jar',
          is_default=False)
+new_fact('jboss.fuse.summary', 'Summary of Fuse info',
```

Comment:
Let's be consistent and say `Summary info of Fuse information`

Reply:
Done